Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Delete.ts

```diff
@@ -20,63 +20,61 @@ export async function deleteDocumentByIdTransaction(
 ): Promise<DeleteResult> {
   const mongoCollection: Collection<MeadowlarkDocument> = getDocumentCollection(client);
```

Comment:
This function only ever uses `client` to get the `mongoCollection`, so better to just pass in a `Collection<MeadowlarkDocument>`